tsconfig.json

```
@@ -4,7 +4,7 @@
     "esModuleInterop": true,
     "jsx": "react-native",
     "lib": ["esnext", "dom"],
-    "types": ["jest"],
+    "types": ["jest", "node"],
```

```suggestion
    "types": ["jest"],
```

Same as in `package.json`, I don't think we should be using `node` types.

---

Done in [32595fb](https://github.com/software-mansion/react-native-gesture-handler/pull/2852/commits/32595fb8a63e99724233921ba9ceb0ccfaf4eeae)